.changes/v3.0.0/714-features.md

```diff
@@ -4,9 +4,9 @@
   `VCDClient.GetTrustedCertificateById`, `TrustedCertificate.Update`, `TrustedCertificate.Delete`
   [GH-714]
 * vCenter management types `VCenter` and `types.VSphereVirtualCenter` adds Create, Update and Delete
- methods: `VCDClient.CreateVcenter`, `VCDClient.GetAllVCenters`, `VCDClient.GetVCenterByName`,
+ methods: `VCDClient.CreateVcenter`,`VCDClient.CreateVcenterAsync`, `VCDClient.GetAllVCenters`, `VCDClient.GetVCenterByName`,
```

Comment:
I actually wouldn't mind a separate changelog entry for this with explanation of the cause for it/its purpose.

Reply:
Changed and added a separate `notes` about the generic function `createInnerEntityAsync`